### PR TITLE
fix(care plan): fix view button label in list

### LINKED ISSUE #2372: Care Plan list view action button shows translation key instead of translation text
🐛 Bug Report

Care Plan list action button shows translation key, **actions.view** instead of translation text **View**

## To Reproduce

Create a Patient
Add a Diagnosis
Add a Care Plan for the Diagnosis you just added
View the list of Care Plans
Notice the view action button has a label that says actions.view instead of View

## Expected behavior

On the Care Plan listing, the view action button should display **View**

## Your Environment

- *node version*: 10
- *fastify version*: >=1.0.0
- *os*: Windows 10
- *any other relevant information*

### PR DESCRIPTION
**Pull request by gaweki**
Fixes #2372

**Changes proposed in this pull request:**

- add t() function invoking on key actions.view(available for enUS translations)

**Newly added dependencies with [Bundlephobia](https://bundlephobia.com/) links:**

NONE

### CODE REVIEW

**Comment by jackcmeyer** [comment]
@gaweki looks like the build is failing. Also, you'll need to sign the CLA.

**Comment by gaweki** [comment]
how to sign the CLA, mr @jackcmeyer ?

**Comment by jackcmeyer** [comment]
@gaweki click the hyperlinks in the comment: https://github.com/HospitalRun/hospitalrun-frontend/pull/2376#issuecomment-688919210

File: src/shared/locales/enUs/translations/labs/index.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default {
     filterTitle: 'Filter by status',
     search: 'Search labs',
     successfullyUpdated: 'Successfully updated',
+    successfullyCompleted: 'Succesfully Completed',
```
Comment (morrme) [inline src/shared/locales/enUs/translations/labs/index.ts:7]: I believe @jackcmeyer  wanted this to be `Successfully completed ${type} for ${patient}`

**Review by jackcmeyer — CHANGES_REQUESTED** [review]
To keep Pull Requests as atomic units, I think we should move the fixing of the lab success message to a different pull request.

**Comment by gaweki** [comment]
> To keep Pull Requests as atomic units, I think we should move the fixing of the lab success message to a different pull request.

i make PR from different branch and start from latest commit on master branch original repo?

File: src/shared/locales/enUs/translations/labs/index.ts (hunk 1 above)
Comment (gaweki) [reply src/shared/locales/enUs/translations/labs/index.ts]: should i add 'for' text? Cause the original text must be `${t('labs.successfullyCompleted')} ${complete.type} ${patient?.fullName} ` @morrme @jackcmeyer

**Comment by gaweki** [comment]
> To keep Pull Requests as atomic units, I think we should move the fixing of the lab success message to a different pull request.

already reset commits. another commit is on another  PR @jackcmeyer

**Review by blestab — APPROVED** [review]

**Review by jackcmeyer — APPROVED** [review]

**Pull request merged**